cub/docs/benchmarking.rst

```diff
@@ -0,0 +1,53 @@
+Running CUB Benchmarks
+*************************************
+
+This file contains instrutions on how to run all CUB benchmarks using CUB tuning infrastructure.
+
+.. code-block:: bash
+
+    pip3 install --user fpzip pandas scipy
+    git clone https://github.com/NVIDIA/cccl.git
+    mkdir build; cd build
+    cmake .. -DCCCL_ENABLE_THRUST=OFF\
```

Comment:
suggestion: We should rather tell cmake to create the build dir as we do in the ci scripts
```suggestion
    cmake -S .. -B build
             -DCCCL_ENABLE_THRUST=OFF\
```

Reply:
Note (non-blocking): let's add presets for CUB benchmarks as part of https://github.com/NVIDIA/cccl/pull/324. 

In an ideal future world, we won't need anyone to do explicit cmake invocations with anything other than a particular preset (and maybe GPU archs).